Used orthophonology for Old Norse

clemsciences opened the pull request:
Reformated some files about orthophonology. Made an Old Norse orthophonology module with tests.

kylepjohnson commented:
@clemsciences thanks you for the PR.

A things to points/questions:
* `cltk/phonology/old_norse/orthophonology.py`: @clemsciences this is an entirely new module, while the other changes are mostly formatting. Ideally, I would like to see two pull requests, one for the formatting and another for the addition of the module. The reason why is that your formatting changes are probably unobjectionable, while a new module could take days or weeks more of study and edits.
* `cltk/phonology/old_norse/orthophonology.py`: @free-variation out of curiosity, is there a way of writing tests for a new addition, like this? For example, let's say someone wanted to make a change to Clement's rules … how would we judge whether the change was preferable? This is not a huge concern right now, just wondering.
* `cltk/phonology/orthophonology.py`: @free-variation A few non-trivial changes to your code, so I will let you review and work this out with Clement. Of course, bring me in if you want a third opinion.

clemsciences commented:
#923 and #924 replace this pull request. I absolutely have not changed the core code of @free-variation, it is just a question of spaces. I also added two methods and renamed one.

kylepjohnson commented:
Thanks, Clem. I would have accepted this PR as-is, but the two help :-)
